client/src/feature/userReducer.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const userSlice = createSlice({
+  name: 'user',
+  initialState: {
+    userInfo: {
+      name: '', id: '', isAdmin: false,
+    },
+  },
+  reducers: {
+    signUp: (state, action) => {
+      state.userInfo = action.payload;
+    },
+  },
+
+});
```

Comment:
Create a folder called redux, inside it put the app and the feature, inside the feature create another folder called user and put this file inside it, but rename it to userSlice.